libcxx/docs/Hardening.rst

```diff
@@ -155,13 +155,13 @@ easier to reason about the high-level semantics of a hardening mode.
   in the library -- whatever the consequences are, they will happen in the user
   code.
 
-- ``pedantic`` -- checks preconditions that are imposed by the Standard, but
-  violating which happens to be benign in libc++.
+- ``pedantic`` -- checks preconditions that are imposed by the C++ standard,
```

Comment:
I'm not sure I'm a fan of changing "the Standard" to "the C++ standard". There aren't that many standards a C++ standard library could refer to.